pkg/controller/clusterclaim/clusterclaim_controller_test.go

```diff
@@ -208,6 +221,35 @@ func TestReconcileClusterClaim(t *testing.T) {
 			}},
 			expectAssignedClusterDeploymentDeleted: true,
 		},
+		{
+			name:  "deleted-pending cluster",
```

Comment:
`cluster marked for deletion` makes more sense

Reply:
✓